polar-core/src/polar.rs

```diff
@@ -249,6 +246,13 @@ impl Polar {
             kb.clear_rules();
             return Err(e);
         }
+
+        // TODO(@gkaemmer) is this the right place for this?
+        let mut warnings = vec![];
+        warnings.append(&mut check_no_allow_rule(&*kb)?);
```

Comment:
I think this deref will happen automatically

```suggestion
        warnings.append(&mut check_no_allow_rule(&kb)?);
```